.github/workflows/invoke-platforms-lambda-function.yml

```diff
@@ -1,23 +1,26 @@
 name: invoke-platforms-lambda-function
 on:
   workflow_dispatch:
-  
+
 jobs:
   invoke:
     runs-on: ubuntu-latest
-    
+
     defaults:
       run:
         working-directory: platforms/lambda
-        
+
     env:
       AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       AWS_ROLE: ${{ secrets.AWS_ROLE }}
-      
+
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
-      - run: yarn
+        with:
+          cache: 'yarn'
+      - name: Install Dependencies
+        run: yarn install
```

Comment:
difference between just `yarn` and `yarn install`?

Reply:
Just having one explicit everywhere, it's the same but easier to search